log_helpers.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/Sirupsen/logrus"
+
+	"github.com/TykTechnologies/tyk/config"
+)
+
+func getLogEntryForRequest(r *http.Request, key string, data map[string]interface{}) *logrus.Entry {
+	// populate http request fields
+	fields := logrus.Fields{
+		"path":   r.URL.Path,
+		"origin": requestIP(r),
+	}
+	// add key to log if configured to do so
+	if key != "" && config.Global.EnableKeyLogging {
+		fields["key"] = key
```

Comment:
Looks good! But I think it makes sense make it obvious to the user that tokes are hidden. Smth like this:
```
if key != "" {
    fields["key"] = key

    if !config.Global.EnableKeyLogging {
      key = "<hidden>"      
    }
}
```